### ISSUE TITLE
Tests fail with perl 5.27.8

### ISSUE THREAD

**Issue by eserte** [hoelzro/devel-trace-syscall#19]
See subject. A sample fail report: http://www.cpantesters.org/cpan/report/c0cf527e-feb6-11e7-9d0c-feba83bfd83b

The most interesting part is probably
```
# | 3| args => [ | args => [ |
# | 4| '"/dev/null"', | '"/dev/null"', |
# * 5| '0x80241', | '0x241', *
# | 6| '0666' | '0666' |
# | 7| ], | ], |
```
which is probably related to the documented change "Close-on-exec flag set atomically" (see perldelta.pod in perl 5.27.8).

**Comment by hoelzro** [comment]
Thanks for the report  @eserte - it turns out it has something to do with using the `open` or `openat` system call for IO.  I'll need to make the test suite more robust for this.

**Comment by hoelzro** [comment]
This should be fixed as of version 0.05 of the module.
